Source/WebKit/Shared/AuxiliaryProcess.h

```diff
@@ -210,6 +210,7 @@ struct AuxiliaryProcessInitializationParameters {
 #if PLATFORM(COCOA)
     SDKAlignedBehaviors clientSDKAlignedBehaviors;
 #endif
+    bool lockdownModeEnabled { false };
```

Comment:
WebKit boolean variables need a prefix, per coding style.

Reply:
That is a good point. The other process types does not need this flag. I will update the PR.

Thanks for reviewing!